client/ayon_core/tools/publisher/widgets/widgets.py

```diff
@@ -1602,8 +1629,18 @@ def set_current_instances(self, instances, context_selected):
 
                 if multivalue:
                     widget.set_value(values, multivalue)
+                    is_default = True
+                    for value in values:
+                        is_default = value == attr_def.default
+                        if not is_default:
+                            break
                 else:
                     widget.set_value(values[0])
+                    is_default = values[0] == attr_def.default
+                if is_default:
+                    label_widget.setStyleSheet("")
+                else:
+                    label_widget.setStyleSheet("color: #4287f5")
```

Comment:
Looking at the code duplication of this stylesheet value I wonder if we should instead implement this:

```python
set_styled_property(widget, property, value):
    """Set widget property and refresh dynamic style"""
    if widget.property(property) == value:
        return
    widget.setProperty(property, value)
    # force stylesheet refresh
    style = widget.style()
    style.unpolish(widget)
    style.polish(widget)
``` 

According to what I read online that should work with e.g. styling as:
```python
from PySide2 import QtWidgets, QtCore

app = QtWidgets.QApplication.instance()
app.setStyleSheet("*[nondefault=true] { font: bold }")


button = QtWidgets.QPushButton("hello")
button.show()

timer = QtCore.QTimer()
timer.setInterval(1000)


def set_styled_property(widget, property, value):
    """Set widget property and refresh dynamic style"""
    if widget.property(property) == value:
        return
    widget.setProperty(property, value)
    # force stylesheet refresh
    style = widget.style()
    style.unpolish(widget)
    style.polish(widget)

def toggle_property():
    new_state = not bool(button.property("nondefault"))
    print(f"Setting state: {new_state}")
    set_styled_property(button, "nondefault", new_state)

timer.timeout.connect(toggle_property)
timer.start()
```
This shows that it does work, even if the stylesheet itself exists globally and not on the object itself. (Note that you usually don't want to apply it on the application, but this is just example code).

By doing it this way we can adjust the more global AYON stylesheet to have e.g.

```css
#PublishWindow *[nondefault=true] { font: bold }
```